### DISCUSSION TITLE
OMEdit Sensitivity Analysis and optimization not working, missing libraries

### DISCUSSION THREAD

**Discussion by fhi2023** [OpenModelica/OpenModelica#11039] [General]
Hello folks,

I have some questions with regards to the integrated sensitivity and optimization module of OMEdit.
When trying to solve my model, which lives in my own package, but utilizes some Modelica standard components, I face these errors:

"Error: Failed to load package Modelica (default) using MODELICAPATH C:/Users/XXX/SES_EL"
--> This is when I set the MODELICAPATH to the path of my own package

"Error: Failed to insert class CreepTest_BG within SES_EL.Tests;

 the available classes were:
  Modelica.Units.Icons.Conversion
  Modelica.Units.Icons
  ..."
-->This is when I keep Modelica Path unchanged. The tool seems to be looking for my model in the Modelica Path, but is in a different directory.

I've tried different combinations of MODELICAPATH for the standard lib, my custom lib, both...

Can anyone tell me how to set it up such that you can optimize a model which:
- Lives in your own custom library
- Utilizes Modelica standard components

Thanks very much in advance!

**Comment by casella** [comment]
@mahge, @adrpo, can you help with that?
